bcgov/HOUS-permit-portal
# Feature/bphh 351

## Description

Fix login/logout, manager home, template and requirement block search, refactor grid, normalize i18n keys, search loading spinners, and more

https://hous-bssb.atlassian.net/browse/BPHH-351?atlOrigin=eyJpIjoiNjk2OTEzY2NiZDQwNDZiOGE0NWQ0OTM0MTZjYzJmM2MiLCJwIjoiaiJ9

## What type of PR is this? (check all applicable)

- [ x] 🍕 Feature

## Related Tickets & Documents

https://hous-bssb.atlassian.net/browse/BPHH-351?atlOrigin=eyJpIjoiNjk2OTEzY2NiZDQwNDZiOGE0NWQ0OTM0MTZjYzJmM2MiLCJwIjoiaiJ9

## Steps to QA

log in asreview_manager
Check out the template and library screens.
try login and logout
See how jurisdiction is loaded automagically

## UI Accessibility Checklist

_Note the following checklist is not an exhaustive list. Check
the [WebAIM checklist](https://webaim.org/standards/wcag/checklist) for
a user-friendly checklist with more details. Aim to meet at least *AA* conformance level where applicable. Check
the [WCAG guidelines](https://webaim.org/standards/wcag/checklist) for the full guidelines._

- [x ] Markup uses semantic HTML?
- [x ] Additional context added through `aria-roles` and `aria-labels`?
- [ x] Checked
      with [Wave Browser Extension](https://chromewebstore.google.com/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh)
      for accessibility errors?
- [ x] Usable with a keyboard and navigable in a logical order?
- [ x] Usable using a screen reader (e.g. Voiceover for Mac) with enough context for the user?

## General Checklist
